ci/build-test-qemu.yaml

```diff
@@ -0,0 +1,37 @@
+job:
+  strict: true
+recipe:
+  git_ref: master
+  git_url: https://github.com/openshift/os
+  repos:
+  - url: http://base-4-8-rhel84.ocp.svc.cluster.local
+stages:
+# Execution Orders are groupings of pods. All pods in each execution order
+# will be run at the same time as soon as their depdencies are ready.
+- id: Base build
+  execution_order: 1
+  build_artifacts:
+   - base
+   - extensions
+
+# These tests will be executed in parallel in seperate pods
+- id: Test Basic
+  execution_order: 2
+  commands:
+  - cosa kola run --qemu-nvme=true basic
+  require_artifacts:
+  - qemu
```

Comment:
Since gangplank is cosa specific, why not have a higher level verb like:
```
kola_run: --qemu-nvme=true basic
````
and that knows that the qemu artifact is required or so?

Reply:
I want to eventually add that verb -- I think it would be really nice -- for now, though, getting Gangplank in production use is the priority.